protocols/kad/src/behaviour.rs

```diff
@@ -180,6 +182,8 @@ impl Default for KademliaConfig {
             connection_idle_timeout: Duration::from_secs(10),
             kbucket_inserts: KademliaBucketInserts::OnConnected,
             caching: KademliaCaching::Enabled { max_peers: 1 },
+            // By default, a peer will not be in client mode.
+            client: Mode::default(),
```

Comment:
```suggestion
            client: Mode::default(),
```

Just in case we ever update the default, we won't have an outdated comment.